### DISCUSSION TITLE
/roborock-s5-zoned-cleanup-with-valetudo-home-assistant/

### DISCUSSION THREAD

**Discussion by migratecomments** [ironicbadger/blog.ktz.me#49] [Announcements]
Roborock S5 zoned cleanup with Valetudo + Home Assistant
        
[https://blog.ktz.me/roborock-s5-zoned-cleanup-with-valetudo-home-assistant/](https://blog.ktz.me/roborock-s5-zoned-cleanup-with-valetudo-home-assistant/)

**Comment by migratecomments** [comment]
```header
Originally posted by lurulf on 2020-11-04 02:36
```

Thanks Alex. Do I have to enable the "perisent map data" in the valetudo UI? I assumed I had to, but after some cleanups with this enabled, the map was really messed up with overlays misaligned/rotated. I think this relates to that I lifted the vacuum by hand when it was stuck. Any thoughts? -Frode

**Comment by Surfer2010** [comment]
hello, 
nice help but i don't get the sensor information to show? stats of cleaning main brush and so on. what is the trick?
